benchexec/tools/ceagle_absref.py

```diff
@@ -1,55 +0,0 @@
-# coding=utf-8
```

Comment:
This variant of ceagle has been integrated into our main tool, and the old one are not maintained any more.